module/input/NatNet/src/NatNet.cpp

```diff
@@ -173,12 +173,12 @@ namespace module::input {
         mocap->timecode    = ReadData<uint32_t>::read(ptr, version);
         mocap->timecodeSub = ReadData<uint32_t>::read(ptr, version);
 
-        // In version 2.9 timestamp went from a float to a double
+        // In version 2.9 time went from a float to a double
         if (version >= 0x02090000) {
```

Comment:
What even is this if-else??

Reply:
It checks the version and reads in different data?

Reply:
Actually no this makes sense now, I thought it was doing something else.